src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/GoogleDataprocDAO.scala

```diff
@@ -116,25 +117,25 @@ class GoogleDataprocDAO(protected val dataprocConfig: DataprocConfig, protected
 
     // Create a Cluster and give it a name and a Cluster Config
     val cluster = new GoogleCluster()
-      .setClusterName(clusterName)
+      .setClusterName(clusterName.string)
       .setConfig(clusterConfig)
 
     // Create a dataproc create request and give it the google project, a zone, and the Cluster
-    val request = dataproc.projects().regions().clusters().create(googleProject, dataprocConfig.dataprocDefaultRegion, cluster)
+    val request = dataproc.projects().regions().clusters().create(googleProject.string, dataprocConfig.dataprocDefaultRegion, cluster)
 
-    executeGoogleRequestAsync(googleProject, clusterName, request)  // returns a Future[DataprocOperation]
+    executeGoogleRequestAsync(googleProject, "Cluster " + clusterName.string, request)  // returns a Future[DataprocOperation]
```

Comment:
Wonder if we could just use the case class `toString` here? A trick: a case class toString prefix can be customized like this:
```
scala> case class ClusterName(string: String) {
     | override def productPrefix = "Cluster: "
     | }
defined class ClusterName

scala> ClusterName("my-cluster").toString
res0: String = Cluster: (my-cluster)
```

Reply:
oh that is cool, thanks